clearwater-infrastructure/usr/share/clearwater/bin/list-subscribers

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# Copyright (C) 2016 Metaswitch Networks Ltd. All rights reserved.
+
+import sys
+import requests
+import argparse
+from configobj import ConfigObj
+
+
+parser = argparse.ArgumentParser(description='Lists the subscribers currently '
+                                             'assigned to this S-CSCF.')
+args = parser.parse_args()
+
+
+CONFIG_FILE = '/etc/clearwater/shared_config'
+API_KEY = 'hs_mgmt_hostname'
```

Comment:
Can we use a better name for this variable (HOSTNAME_KEY or something?). Seeing `config.get(API_KEY)` made me think we were trying to get an API key for authentication.